.github/workflows/test.yml

```diff
@@ -240,6 +240,12 @@ jobs:
         run: |
           pip install --requirement requirements-full.txt
 
+      # Is there a better way to do this?
```

Comment:
Can you add "# TODO: Is there a better.." that way we can search in the code for that keyword and see what there is to do